backend/src/main/java/com/hackner/musicompass/model/RecommendationTags.java

```diff
@@ -1,7 +1,20 @@
 package com.hackner.musicompass.model;
 
-import lombok.Data;
+import lombok.*;
+
+import java.util.Date;
+import java.util.List;
 
 @Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
 public class RecommendationTags {
+
+    private Boolean recommended;
```

Comment:
What is the purpose of this Boolean? 
I can only see you setting it to true and never to false (and if it is posted from the frontend it still will be set to true?!)

Reply:
This boolean marks, if the artist is set as a favourite/ a recommended artist. If the status is later on changed, the boolean will be changed to false.